src/features/Workout/SectionDetail.js

```diff
@@ -346,21 +348,51 @@ const SectionDetail = () => {
                   </span>
                   <span className="text-sm">Weight Choosing Guide</span>
                 </div>
-                <div>
-                  <h2 className="workout-gradient-text my-4 text-2xl">
-                    Data Inputs
-                  </h2>
-                  {dataInput.map((input, index) => (
-                    <DataInputComponent2
-                      key={index}
-                      inputId={input.id}
-                      inputType={input.type}
-                      inputOptions={input.options}
-                      placeholder={input.label}
-                      label={input.label}
-                    />
-                  ))}
-                </div>
+
+               <div>
+        <h2 className="workout-gradient-text my-4 text-2xl">
+          Data Inputs
+        </h2>
+        {dataInput.slice(0, 2).map((input, index) => (
+          <DataInputComponent2
+            key={index}
+            inputId={input.id}
+            inputType={input.type}
+            inputOptions={input.options}
+            placeholder={input.label}
+            label={input.label}
+          />
+        ))}
+        
+        {name === "Gymnastic Strength" && dataInput.length > 2 && (
```

Comment:
can we set the condition based on code of the workout section?

Reply:
Good Point, I can use the code instead of the name.